blocks/headline-block/features/headline/default.test.jsx

```diff
@@ -27,14 +27,16 @@ describe('the headline feature for the default output type', () => {
       const wrapper = mount(<Headline />);
 
       expect(wrapper.find('h1')).toHaveClassName('headline');
+      // checking for styled component class
       expect(wrapper.find('h1').hasClass(/sc-/)).toBe(true);
     });
 
     it('should dangerously set the innerHTML to the headline content', () => {
       const { default: Headline } = require('./default');
-      const wrapper = shallow(<Headline />);
+      const wrapper = mount(<Headline />);
```

Comment:
I occasionally worry about using mount because it tends it falsely increase coverage when running tests. Meaning, it'll mount ALL the things, and even though things aren't asserted, it still has run the code and considers itself covered.

I prefer to use shallow and if I *need* to mount a child node, I try and use the `dive` method.

https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/dive.html

I don't think you have to change this, just worth having the conversation around testing methods :D

Reply:
huh, haven't seen this discussion. Yeah good to talk about